dev/NumberBox/NumberBox.cpp

```diff
@@ -575,9 +579,6 @@ void NumberBox::UpdateTextToValue()
         });
         m_textUpdating = true;
         Text(newText.data());
-
-        // This places the caret at the end of the text.
-        textBox.Select(static_cast<int32_t>(newText.size()), 0);
```

Comment:
>textBox.Select(static_cast<int32_t>(newText.size()), 0); [](start = 8, length = 56)

Does this mean we have no method for setting the caret's position without selecting? seems like it might be an API gap on text box.

Reply:
That's correct. Selection and the caret are kind of the same thing, so I get it, but it is a little hidden if you're just looking to set caret position.

Reply:
Does that mean you don't think there is an API gap on text box? I'm wondering if a feature proposal is warranted